Makefile

```diff
@@ -13,12 +13,10 @@ else
 endif
 
 coverage-report: ## coverage-report: Run the unit tests with code coverage enabled.
-	unset FT_NEXT_BACKEND_KEY && istanbul cover node_modules/.bin/_mocha --report=$(if $(CIRCLECI),lcovonly,lcov) 'test/**/*.test.js'
+	unset FT_NEXT_BACKEND_KEY && FT_GRAPHITE_KEY=foobar istanbul cover node_modules/.bin/_mocha --report=$(if $(CIRCLECI),lcovonly,lcov) 'test/**/*.test.js'
```

Comment:
I've introduced the `n-health` module which blows up if that's not present:

![screen shot 2017-05-11 at 09 48 13](https://cloud.githubusercontent.com/assets/4750238/25940734/00a5a782-362f-11e7-85a0-9740477b23c9.png)